lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -609,13 +610,18 @@ def _section_send_email(course, access):
         # xblock rendering.
         request_token=uuid.uuid1().get_hex()
     )
+    cohorts = []
+    if is_course_cohorted(course_key):
+        cohorts = [cohort for cohort in get_course_cohorts(course)]
```

Comment:
The purpose of the comprehension was to turn a list of `CourseUserGroup` into a list of `CourseCohort` (as they're slightly different classes connect via a 1-to-1 field). Now that you mention it though, that may not be needed either. I'll investigate.

`get_course_cohorts` will definitely return a list of cohorts that exist in the database for a course that has cohorts disabled, so the check is needed here.